neural-networks/speech-recognition/whisper-tiny-en/oakapp.toml

```
@@ -1,15 +1,15 @@
 identifier = "com.luxonis.whisper"
+app_version = "1.0.0"
 
 prepare_container = [
     { type = "RUN", command = "apt-get update" },
-    { type = "RUN", command = "apt-get install -y wget python3 python3-pip ffmpeg libglib2.0-0 libgl1-mesa-glx" },
-    { type = "RUN", command = "pip3 install depthai==3.0.0-alpha.6.dev0+4b380c003bbfe52348befdb82cf32013a7db2793 --break-system-packages --extra-index-url https://artifacts.luxonis.com/artifactory/luxonis-python-snapshot-local/" },
-    { type = "RUN", command = "wget -O /libusb-1.0.so https://rh-public.luxonis.com/libusb/1.0.26/libusb-x86_64.so" },
-    { type = "RUN", command = "cp /libusb-1.0.so /usr/local/lib/python3.11/dist-packages/depthai.libs/libusb-1-105e664f.0.so" },
+    { type = "RUN", command = "apt-get install -y python3 python3-pip ffmpeg git libglib2.0-0 libgl1-mesa-glx wget xcb libportaudio2 libportaudiocpp0 portaudio19-dev" },
+   
 ]
 
 prepare_build_container = []
 
-build_steps = ["pip3 install openai-whisper==20231117 --no-deps --break-system-packages", "pip3 install -r /app/requirements.txt --break-system-packages"]
+build_steps = ["pip3 install -r /app/requirements.txt --break-system-packages"]
 
-entrypoint = ["bash", "-c", "python3 -u /app/main.py --device_ip '' --audio_file /app/assets/audio_files/command_LED_yellow.mp3"]
\ No newline at end of file
+
+entrypoint = ["bash", "-c", "python3 -u /app/main.py --device_ip '' --audio_file /app/assets/audio_files/command_LED_yellow.mp3"]
```

Can't we have just: 
```suggestion
entrypoint = ["bash", "-c", "python3 -u /app/main.py"]
```
because other parameters are just set to defaut no?

---

But we can set one of the prerecorded files as default in the arguments init right? E.g. the "yellow" mp4

---

Then it will always first run the prerecorded clip, even if the user wants to use just record feature.